Dockerfiles/cluster-agent/Dockerfile

```diff
@@ -12,10 +12,10 @@ LABEL org.opencontainers.image.source "https://github.com/DataDog/datadog-agent"
 WORKDIR /output
 
 COPY datadog-cluster-agent.$TARGETARCH opt/datadog-agent/bin/datadog-cluster-agent
-COPY ./conf.d etc/datadog-agent/conf.d
+COPY --chmod=644 ./conf.d etc/datadog-agent/conf.d
 COPY ./datadog-cluster.yaml etc/datadog-agent/datadog-cluster.yaml
```

Comment:
should we do also the same change here?

Reply:
I think we should apply it to every `COPY` instruction but for the GBI checks to pass, I just need to apply it on this line.

Reply:
Maybe we can modify all the `COPY` instructions to `COPY --chmod` in another PR. I suppose it could break things so let's be prudent.

Reply:
After a discussion, I will create a JIRA card to modify all the `COPY` instructions.